packages/backend-core/src/redis/invite.ts

```diff
@@ -0,0 +1,96 @@
+import { redis, utils, tenancy } from "../"
+import env from "../environment"
+
+const TTL_SECONDS = 60 * 60 * 24 * 7
+
+interface Invite {
+  email: string
+  info: any
+}
+
+interface InviteWithCode extends Invite {
+  code: string
+}
+
+let client: redis.Client
+
+async function getClient(): Promise<redis.Client> {
+  if (!client) {
+    client = new redis.Client(redis.utils.Databases.INVITATIONS)
+    await client.init()
+  }
+  return client
+}
```

Comment:
This Redis client used to get set up in an explicit `init()` function, and torn down in an explicit `shutdown()` function that were part of the `packages/worker` start up and shutdown code. I'm lazily initialising them here and not shutting them down, depending instead on the process death to get rid of them. Not sure if this is problematic.

Reply:
Generally a quite like lazy initing of resources like this - the services will always confirm that the redis parameters are correct on startup, so therefore it'll be safe for this connection to occur when needed.